PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,7 +4,7 @@
 
 ### PR Overview
 
-- [ ] This PR requires new unit tests [y/n] (make sure tests are included)
-- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
-- [ ] This PR is backwards compatible [y/n]
-- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)
+- [n] This PR requires new unit tests [y/n] (make sure tests are included)
```

Comment:
Please revert, those should be markdown checkboxes
[x] or [ ]

Reply:
ok got it